docs/data-sources.md

```diff
@@ -21,11 +21,11 @@ With its coverage of roads, POIs, landuse, as well as rich metadata, OpenStreetM
 
 All OpenStreetMap data is licensed under the [ODbL](http://opendatacommons.org/licenses/odbl/), a [share-alike](https://en.wikipedia.org/wiki/Share-alike) license which also requires attribution.
 
-## openstreetmapdata.com
+## osmdata.openstreetmap.de
 
-`source:openstreetmapdata.com`
+`source:osmdata.openstreetmap.de`
 
-We include coastline-derived water polygons from [openstreetmapdata.com](http://openstreetmapdata.com) at mid- and high-zooms. This service was created by Jochen Topf and Christoph Hormann for the OpenStreetMap community and the general public and it rocks!
+We include coastline-derived water polygons from [osmdata.openstreetmap.de](https://osmdata.openstreetmap.de) at mid- and high-zooms. This service was created by Jochen Topf and Christoph Hormann (now run by [FOSSGIS e.V.](https://www.fossgis.de/)) for the OpenStreetMap community and the general public and it rocks!
```

Comment:
We should say this new source replaces the old `openstreetmapdata.com` source?

Reply:
Sure, fixed in ac65de3.